package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "lint:lockfile": "lockfile-lint --path package-lock.json --type npm --allowed-hosts npm --validate-https --validate-integrity",
-    "unit": "c8 -o ./coverage/unit tap --expose-gc --jobs=1 --no-coverage tests/unit/*.tap.js",
+    "unit": "c8 -o ./coverage/unit node --expose-gc --test tests/unit/*.test.js",
```

Comment:
I was going to wait until you brought this out of draft, but I see you're trying to make this work. Unfortunately, we cannot rely on shell expansion of wildcards and Node doesn't support globbing until 20. So we need to use `borp` to run tests. See how we do it in the agent repo.

Reply:
in this case it's fine though. there are only tests at the root, it runs them

Reply:
I tried `"unit": "c8 -o ./coverage/unit node --expose-gc --test tests/unit/*.test.js",` and it didn't work. So I tried it with `borp` and it worked.